### PR TITLE
Update PublicApiGenerator to 9.3.0

### PR DESCRIPTION
**Pull request by blairconrad**
Supports #1613.

### CODE REVIEW

**Comment by blairconrad** [comment]
Nullable types come with an attribute on the type, and old PublicApiGenerator could not deal with them.

File: tests/FakeItEasy.Tests.Approval/ApiApproval.ApproveApi40.approved.txt [hunk 1]
```diff
@@ -432,16 +426,15 @@ namespace FakeItEasy.Configuration
...
+    public interface IAnyCallConfigurationWithNoReturnTypeSpecified : FakeItEasy.Configuration.IArgumentValidationConfiguration<FakeItEasy.Configuration.IVoidConfiguration>, FakeItEasy.Configuration.IAssertConfiguration, FakeItEasy.Configuration.ICallBaseConfiguration<FakeItEasy.Configuration.IVoidConfiguration>, FakeItEasy.Configuration.ICallbackConfiguration<FakeItEasy.Configuration.IVoidConfiguration>, FakeItEasy.Configuration.IDoNothingConfiguration<FakeItEasy.Configuration.IVoidConfiguration>, FakeItEasy.Configuration.IExceptionThrowerConfiguration<FakeItEasy.Configuration.IVoidConfiguration>, FakeItEasy.Configuration.IOutAndRefParametersConfiguration<FakeItEasy.Configuration.IVoidConfiguration>, FakeItEasy.Configuration.IVoidArgumentValidationConfiguration, FakeItEasy.Configuration.IVoidConfiguration, FakeItEasy.Configuration.IWhereConfiguration<FakeItEasy.Configuration.IAnyCallConfigurationWithNoReturnTypeSpecified>, FakeItEasy.IHideObjectMembers
```
Comment (thomaslevesque) [inline tests/FakeItEasy.Tests.Approval/ApiApproval.ApproveApi40.approved.txt]: Not sure why this is highlighted... I don't see any difference here

File: tests/FakeItEasy.Tests.Approval/ApiApproval.ApproveApi40.approved.txt [hunk 2]
```diff
@@ -558,7 +540,7 @@ namespace FakeItEasy.Core
         public FakeCreationException(string message) { }
         public FakeCreationException(string message, System.Exception innerException) { }
     }
-    public class FakeManager : FakeItEasy.Core.IFakeCallProcessor
```
Comment (thomaslevesque) [inline tests/FakeItEasy.Tests.Approval/ApiApproval.ApproveApi40.approved.txt]: Never noticed this before... `IFakeCallProcessor` is internal, so it shouldn't show up here. Well, apparently it's been fixed!

File: tests/FakeItEasy.Tests.Approval/ApiApproval.ApproveApi40.approved.txt [hunk 3]
```diff
@@ -597,12 +579,15 @@ namespace FakeItEasy.Core
         void OnAfterCallIntercepted(FakeItEasy.Core.ICompletedFakeObjectCall interceptedCall);
         void OnBeforeCallIntercepted(FakeItEasy.Core.IFakeObjectCall interceptedCall);
     }
-    public class Raise<TEventArgs> : FakeItEasy.Core.IEventRaiserArgumentProvider
-        where TEventArgs : System.EventArgs { }
+    public class Raise<TEventArgs>
+        where TEventArgs : System.EventArgs
+    {
+        public static System.EventHandler<TEventArgs> op_Implicit(FakeItEasy.Core.Raise<TEventArgs> raiser) { }
+        public static System.EventHandler op_Implicit(FakeItEasy.Core.Raise<TEventArgs> raiser) { }
```
Comment (thomaslevesque) [inline tests/FakeItEasy.Tests.Approval/ApiApproval.ApproveApi40.approved.txt]: Also something I didn't notice before. Operators were not included. Another fix, nice!

File: tests/FakeItEasy.Tests.Approval/ApiApproval.ApproveApi40.approved.txt (hunk 1 above)
Comment (blairconrad) [reply tests/FakeItEasy.Tests.Approval/ApiApproval.ApproveApi40.approved.txt]: `ICallBaseConfiguration` and `ICallBackConfiguration` have swapped positions.

**Comment by blairconrad** [comment]
Thanks, @thomaslevesque. Yeah, I like the new fixes too.

**Comment by blairconrad** [comment]
This change has been released as part of [FakeItEasy 5.2.0](https://github.com/FakeItEasy/FakeItEasy/releases/tag/5.2.0).

**Pull request merged**